### ISSUE TITLE
darknet.py fails to load network successfuly if darknet file paths contains white space. (Similar to #40)

### ISSUE THREAD

**Issue by ChilledFerrum** [hank-ai/darknet#43]
This issue is similar to [issue #40 ](https://github.com/hank-ai/darknet/issues/40#issue-2082510069)

When attempting to load a network using darknet.py using paths containing white space, an error is raised

![image](https://github.com/hank-ai/darknet/assets/62751542/a945a5af-bd50-45a4-82d2-7b4c7451531a)
The specific error is triggered by the darknet.data file containing the path to the names folder.
Using double quotes in darknet.data does not fix the error either, in-fact it includes the quotation marks too. 
It seems like an internal function get_paths() in src-lib/data.cpp raises this error by trimming the white spaces from the path.

Input example:
```python
# Dynamically locates Darknet Projects folder located in home/user/Documents/Darknet Projects/...
home_dir = os.path.expanduser("~")
documents_dir = os.path.join(home_dir, "Documents")
darknet_pth = os.path.join(documents_dir, "Darknet Projects/")

yolo_weights_pth = darknet_pth + str(yolo_weights)
config_pth = darknet_pth + str(yolo_weights).split('.')[0] + ".cfg"
data_pth = darknet_pth + str(yolo_weights).split('.')[0] + ".data"
model, class_names, class_colors = load_network(config_pth, data_pth, yolo_weights_pth)
```
If any of the paths config_pth, data_pth, yolo_weights_pth contains white space it'll fail at any circumstance.

Expected output:
Successfully load the network in any cases even if the path contains white space.

Considering this error is occurs in two different cases darknet.py and when using the CLI tool as mentioned in [issue #40 ](https://github.com/hank-ai/darknet/issues/40#issue-2082510069), it's a low-level issue raised in src-lib/data.cpp.

Please look into it.

**Comment by stephanecharette** [comment]
As part of the fix for issue #40, a number of functions were modified or re-written to better support parsing config lines that have embedded spaces.  Including the function that parses the .data file.

So while I wasn't working specifically on this ticket this evening, it is possible that the fixes made will have fixed this issue as well.  Please re-test and let me know if this can be closed, or if this will require more debugging and further modifications.

**Comment by stephanecharette** [comment]
Closing.  Please re-open if this is still an issue.
